ruby/Makefile

```diff
@@ -0,0 +1,13 @@
+version:
+	@cat VERSION
+
+version-set:
+	@echo $(NEW_VERSION) > VERSION
+
+release-prepare: version-set
+	@git add VERSION
+	@git commit -m "Prepare to release v$(NEW_VERSION)"
+	@echo "TODO: figure out how to bump ruby version"
```

Comment:
If possible it would nice to invoke a ruby gem here.

https://www.rubydoc.info/gems/semver2/3.4.2

Reply:
thanks for the suggestion! https://github.com/jondot/version_bumper looks even better.

Reply:
Looks unmaintained to me.